pkg/controllers/provisioning/scheduling/scheduling_benchmark_test.go

```diff
@@ -91,6 +96,12 @@ func BenchmarkScheduling5000(b *testing.B) {
 	benchmarkScheduler(b, 400, 5000)
 }
 
+var includeMinValues *bool
+
+func init() {
+	includeMinValues = flag.Bool("minValues", false, "include minValues in NodePool requirement")
+}
```

Comment:
nit: feels weird that we would keep it as pointer. Can we change it to just be a primitive so there's no funky behavior with pointers and nil references? 
```suggestion
var includeMinValues bool

func init() {
	includeMinValues = lo.FromPtr(flag.Bool("minValues", false, "include minValues in NodePool requirement"))
}
```

Reply:
We have it defaulted to `false` in the assignment, so ideally it should not be having a `nil` reference at any point of time. Anyways, changed it to primitive by making use of another method `flag.BoolVar()` that particularly helps in this case of passing in the flags since `lo.FromPtr()` wouldn't work.